elf2flt.c

```diff
@@ -149,6 +149,17 @@ const char *elf2flt_progname;
 #define O_BINARY 0
 #endif
 
+#if defined(HAVE_BFD_SECTION_SIZE_MACRO_HAS_ONE_ARG)
+#define elf2flt_bfd_section_size(abs_bfd, s) bfd_section_size(s)
```

Comment:
the point of the API change is that the bfd isn't actually used by any of the bfd_section funcs.  so we should structure our API similarly.

```c
#define elf2flt_bfd_section_size(s) bfd_section_size(s)
#else
#define elf2flt_bfd_section_size(s) bfd_section_size(NULL, s)
```